src/components/Table/Example.md

```diff
@@ -417,3 +417,26 @@ const data = [{
 
 <Table data={data} groupBy="name" />
 ```
+
+** onSort Callback **
+
+Providing an `onSort` callback allows you to do additional data fetching if the user invokes sorting on your table.
```

Comment:
Not sure if this description refers to the actual example provided below.  🤔

Reply:
Updated the description to be more generic